src/persistence/settingsserializer.cpp

```diff
@@ -251,10 +262,10 @@ void SettingsSerializer::save()
             writeStream(stream, a.name.toUtf8());
             writeStream(stream, vuintToData(a.size));
 
-            for (uint64_t vi : a.values)
+            for (int vi = 0; vi < a.values.size(); vi++)
```

Comment:
This change seems wrong again, it's not the same meaning.
And why bother changing from a modern range-for to a for(;;) anyway? It's longer and 'm not convinced it's more readable.